src/Visitors/SetPropertyValue.php

```diff
@@ -1,38 +1,86 @@
 <?php
 
-namespace Ronasit\Larabuilder\Visitors;
+namespace RonasIT\Larabuilder\Visitors;
 
+use PhpParser\Modifiers;
 use PhpParser\Node\ArrayItem;
 use PhpParser\Node\Expr\Array_;
 use PhpParser\Node\Expr\ConstFetch;
 use PhpParser\Node\Identifier;
 use PhpParser\Node\Name;
 use PhpParser\Node\PropertyItem;
+use PhpParser\Node\Stmt\Class_;
+use PhpParser\Node\Stmt\Property;
 use PhpParser\Node\Scalar\Float_;
 use PhpParser\Node\Scalar\Int_;
 use PhpParser\Node\Scalar\String_;
-use PhpParser\Node\Stmt\Property;
 use PhpParser\NodeVisitorAbstract;
 use PhpParser\Node;
+use RonasIT\Larabuilder\Enums\ModifierEnum;
 
 class SetPropertyValue extends NodeVisitorAbstract
 {
     public function __construct(
         protected string $name,
         protected mixed $value,
+        protected ?ModifierEnum $accessModifier = null,
     ) {
     }
 
     public function enterNode(Node $node): void
     {
-        if ($node instanceof Property && $node->props[0]->name->name === $this->name) {
-            list($value, $type) = $this->getPropertyValue($this->value);
+        if ($node instanceof Class_) {
+            $stmtsCount = count($node->stmts);
+
+            foreach ($node->stmts as $index => $statement) {
+                $classNode = $statement;
+
+                if ($classNode instanceof Property) {
```

Comment:
could we also check the parent node type? If yes - it will allows to prevent the first check for the `class` node type checking